server/routes/journeys/update-plan/case-management/controller.ts

```diff
@@ -0,0 +1,29 @@
+import { NextFunction, Request, Response } from 'express'
+import { SchemaType } from './schemas'
+import { UpdatePlanController } from '../controller'
+
+export class CaseManagementController extends UpdatePlanController {
+  GET = async (req: Request, res: Response) => {
+    res.render('update-plan/case-management/view', {
+      reasonForPlan: res.locals.formResponses?.['reasonForPlan'] || req.journeyData.plan?.reasonForPlan,
+      caseManager: res.locals.formResponses?.['caseManager'] || req.journeyData.plan?.caseManager,
+      recordUuid: req.journeyData.csipRecord!.recordUuid,
+      backUrl: '../update-plan',
+    })
+  }
+
+  checkSubmitToAPI = async (req: Request<unknown, unknown, SchemaType>, _: Response, next: NextFunction) =>
+    this.submitChanges({
+      req,
+      next,
+      changes: {
+        reasonForPlan: req.body.reasonForPlan,
+        caseManager: req.body.caseManager,
+      },
+      successMessage: 'You’ve updated the case management information.',
+    })
```

Comment:
UpdatePlanController.submitChanges has the message content covered, you don't need to pass in `successMessage`

Reply:
No, it doesn't. We pass it through into submitChanges because the text can change depending on what we're updating